app/messages/index.js

```diff
@@ -0,0 +1,27 @@
+import util from 'util'
+import { messageConfig } from '../config/index.js'
+import { MessageReceiver } from 'ffc-messaging'
+import { farmerApply } from './farmer-apply.js'
+
+const handleMessage = async (message, receiver) => {
+  try {
+    console.log('Received message: ', message.body)
```

Comment:
Don't log the message.

It will contain sensitive data such as email addresses and perhaps more.

Reply:
If logging needed, only log cloud event metadata and nothing in `data` property

Reply:
Just put this in initially to make sure the application was receiving the message when sent via Service Bus, but now know that it does so no longer need the logging and have removed it. 👍🏽